docs/guides/errors.md

```diff
@@ -136,7 +136,14 @@ schema = strawberry.Schema(query=Query)
 
 ## Expected errors
 
-If an error is expected then it is often best to express it in the schema. This allows the client to deal with the error in a robust way.
+Handling expected errors is typically approached in one or both of two ways:
+
+1. Return the error as part of the return type
+2. Append the error to the `errors` array
+
+### Returning as part of the return type
+
+It is often best to express expected errors in the schema. This allows the client to deal with the error in a robust way.
```

Comment:
No change to this section, just reorganization.

This may be a personal preference, but I don't think this section with full examples really makes sense to live in Strawberry documentation. This is more of a schema design recommendation with nothing specific to strawberry itself. As long as users understand interfaces and unions in graphql and strawberry, implementation is straightforward, and, it's rather opinionated (as seen by the 6-7 examples in the references at the bottom).

I think it makes more sense, if we want to keep this section, to instead point to the references already linked and optionally point to other pages in strawberry documentation that would help readers understand how the core concepts of the error approaches could be implemented in strawberry.

Reply:
I don't feel strongly about this months later :)